src/compiler/types.ts

```diff
@@ -5166,7 +5166,7 @@ namespace ts {
         IncludesMask = Any | Unknown | Primitive | Never | Object | Union | Intersection | NonPrimitive | TemplateLiteral,
         // The following flags are used for different purposes during union and intersection type construction
         /* @internal */
-        IncludesStructuredOrInstantiable = TypeParameter,

```

Comment:
So we don't need this anymore? Was this just never used?

Reply:
Right, there was code to set the flag, but we never examined it. We must have used it at some point, though I don't recall what for.